pkg/cloud/services/ec2/ami.go

```diff
@@ -40,15 +40,16 @@ const (
 	// The pattern is:
 	// 1. the string value `capa-ami-`
 	// 2. the baseOS of the AMI, for example: ubuntu-18.04, centos-7, amazon-2
-	// 3. the kubernetes version as defined by the packages produced by kubernetes/release, for example: 1.13.0-00, 1.12.5-01
-	// 4. the timestamp that the AMI was built
-	amiNameFormat = "capa-ami-%s-%s-??-??????????"
+	// 3. the kubernetes version as defined by the packages produced by kubernetes/release with or without v as a prefix, for example: 1.13.0, 1.12.5-mybuild.1, v1.17.3
+	// 4. a `-` followed by any additional characters
+	amiNameFormat = "capa-ami-?%s-?%s-*"
```

Comment:
I'm curious about the first `?` - what is that for? The second `?` is for the optional `v` in the version, right?

Reply:
Fat finger mistake, thanks for catching :)

Reply:
Fixed, thanks!

Reply:
@ncdc multi-tasking has me scattered a bit, but the remaining `?` is indeed for the optional `v` in version